examples/text_labels_and_annotations/label_subplots.py

```diff
@@ -0,0 +1,71 @@
+"""
+==================
+Labelling subplots
+==================
+
+Labelling subplots is relatively straightforward, and varies,
+so Matplotlib does not have a general method for doing this.
+
+Simplest is putting the label inside the axes.  Note, here
+we use `.pyplot.subplot_mosaic`, and use the subplot labels
+as keys for the subplots, which is a nice convenience.  However,
+the same method works with `.pyplot.subplots` or keys that are
+different than what you want to label the subplot with.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.transforms as mtransforms
+
+fig, axs = plt.subplot_mosaic([['a)', 'c)'], ['b)', 'c)'], ['d)', 'd)']],
+                              constrained_layout=True)
+
+for label, ax in axs.items():
+    # 10 points in and 5 points down:
+    trans = mtransforms.ScaledTranslation(10/72, -5/72, fig.dpi_scale_trans)
+    ax.text(0.0, 1.0, label, transform=ax.transAxes + trans,
+            fontsize='medium', va='top', fontfamily='serif',
```

Comment:
I think you should call out the alignment specifically in the description, too.
```suggestion
            fontsize='medium', verticalalignment='top', fontfamily='serif',
```